pom.xml

```diff
@@ -6,12 +6,12 @@
 
     <groupId>net.jeqo</groupId>
     <artifactId>Gizmo</artifactId>
-    <version>1.2.1</version>
+    <version>1.2.2</version>
     <packaging>jar</packaging>
 
     <name>Gizmo</name>
 
-    <description>A unique utility plugin by Jeqo.</description>
+    <description>A unique utility plugin by Jeqo</description>
```

Comment:
This change is unwanted as with the repository restructure we will be changing this, if you want to revert this you can. But just keep in mind we will be changing this